tutorials/jmsActiveMQ/src/site/conf/my-app-conf1.xml

```diff
@@ -1,11 +1,28 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <configuration>
-  <property>
-    <name>dt.attr.MASTER_MEMORY_MB</name>
-    <value>1024</value>
-  </property>
-  <property>
-    <name>dt.application.MyFirstApplication.operator.randomGenerator.prop.numTuples</name>
-    <value>1000</value>
-  </property>
-</configuration>
+	<property>
+		<name>dt.operator.fileOut.prop.filePath</name>
+		<value>target/</value>
```

Comment:
Yes, it works. You will see this under /user/dtadmin/target (assuming the current Unix user is dtadmin and hence the current hadoop user is also dtadmin). I have left it as it is and added some explanation in the README file.